docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  app:
+    image: friendsofsymfony/fos-http-cache
```

Comment:
docker-composer should rather reference the local Dockerfile instead of referencing an image on the hub. It would avoid having to build the image and publish it each time you change the Dockerfile (which is a pain when contributing)